apps/prairielearn/src/pages/workspace/workspace.html.ts

```diff
@@ -33,6 +34,19 @@ export function Workspace({
         ${HeadContents({ resLocals, pageNote, pageTitle })}
         <link href="${assetPath('stylesheets/workspace.css')}" rel="stylesheet" />
         ${compiledScriptTag('workspaceClient.ts')}
+        ${resLocals.assessment?.type === 'Exam' && resLocals.assessment_instance_remaining_ms
+          ? html`${compiledScriptTag('examTimeLimitCountdown.ts')}
+            ${EncodedData(
+              {
+                serverRemainingMS: resLocals.assessment_instance_remaining_ms,
+                serverTimeLimitMS: resLocals.assessment_instance_time_limit_ms,
+                serverUpdateURL: `/pl/course_instance/${resLocals.course_instance_id}/assessment_instance/${resLocals.assessment_instance.id}/time_remaining`,
```

Comment:
```suggestion
                serverUpdateURL: `${resLocals.plainUrlPrefix}/course_instance/${resLocals.course_instance_id}/assessment_instance/${resLocals.assessment_instance.id}/time_remaining`,
```

Alternatively, import lib/config and use `${config.urlPrefix}`.